spec/features/audit/allocation_spec.rb

```diff
@@ -104,4 +104,19 @@
     expect(page).to have_select("allocate_to", selected: "No one")
     expect(page).to have_content("1 items unallocated")
   end
+
+  scenario 'Allocate all content within current page', :js do
+    create_list(:content_item, 100)
```

Comment:
Creating a list of 100 seems a bit too much. What about creating 27 items (page size + a couple of them more)?